docs/usage/configuration-options.md

```diff
@@ -313,6 +313,13 @@ e.g. instead of `renovate/{{parentDir}}-`, configure the template part in `addit
     This setting does not change the default _onboarding_ branch name, i.e. `renovate/configure`.
     If you wish to change that too, you need to also configure the field `onboardingBranch` in your global bot config.
 
+## branchPrefixOld
```

Comment:
Is the text still correct now that you made code changes?

Reply:
you are right, we do need to add something about an existing open PR with `branchPrefixOld`.
that said, what we already have is still relevant though